OrbitGl/Timer.h

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_TIMER_H_
+#define ORBIT_GL_TIMER_H_
+
+#include <string>
+
+#include "GlUtils.h"
+#include "OrbitBase/Profiling.h"
+
+class Timer {
```

Comment:
This is only used in `OrbitGl`, there is no reason to move it to `OrbitBase`, `OrbitBase` should be kept as small as possible.

Reply:
As mentioned in my other comment, we already have the code (duplicated) in OrbitBase! But it's hidden in a macro. Why not make it available to all. I'll clean up the timer class in OrbitGl btw, it can be made super lean.